packages/layout/src/components/cards/components/button.tsx

```diff
@@ -33,28 +34,70 @@ export const UnderlinedButton: React.FC<UnderlinedButtonProps> = ({
 	tabIndex,
 	isEditButton,
 	onCollapseCallback,
-	btnRef,
+	buttonRef,
 	isMainHeading = false,
+	notHeading = false,
 }) => {
-	if (typeof onClick === 'undefined') {
-		return (
-			<div className={styles.buttonPlaceholder}>
-				<Flex cfg={{ flex: '0 0 auto', alignItems: 'center' }}>
-					{isMainHeading ? (
-						<H3 cfg={{ fontWeight: 3 }}>{children}</H3>
-					) : (
-						<H4>{children}</H4>
-					)}
-				</Flex>
-			</div>
-		);
-	}
+	const editViewButtonRef = useRef(null);
+
+	const NotClickableButton: React.FC = () => (
```

Comment:
It's a heading, not a button, so better to name it that way. And a heading is non-clickable so you wouldn't need to say that.